### PR TITLE
File Added : Assignment5/6

### PR DESCRIPTION
**Pull request by kalyani1942**

### CODE REVIEW

**Review by somesh6797 — APPROVED** [review]

**Review by somesh6797 — APPROVED** [review]

File: Kalyani/HTML/Assginment1/Assin.html [hunk 1]
```diff
@@ -1,18 +1,14 @@
...
-    </p>
-</body>
-
-</html>
\ No newline at end of file
+	<head>
+	    <meta charset="UTF-8">
+	    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+	    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+	    <title>AssignmentA1_1</title>
```
Comment (dayanandviveki) [inline Kalyani/HTML/Assginment1/Assin.html]: give proper title eg.Heading Tags instred of AssinmentA!_1

File: Kalyani/HTML/Assginment1/Assin2.html [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
-    <p>Web pages can be created and modified by using professional HTML editors. <br> However, for learning HTML we recommend a simple text editor like Notepad (PC) or TextEdit (Mac).</p>
-</body>
-
-</html>
\ No newline at end of file
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>AssignmentA1_2</title>
```
Comment (dayanandviveki) [inline Kalyani/HTML/Assginment1/Assin2.html]: Changes in title give topic name of something meaning full wording

File: Kalyani/HTML/Assginment1/Assin4.html [hunk 3]
```diff
@@ -1,7 +1,14 @@
 <!DOCTYPE html>
 <html>
-<head>    
-<title> <h2>Introduction</h2></title>
-<p>
-I am Kalyani Deshpande<br>
-I am a Software Enginner
+	<head>
+	    <title>
+	        <h2>Introduction</h2>
```
Comment (dayanandviveki) [inline Kalyani/HTML/Assginment1/Assin4.html]: fot title tag don't need heading tag or h2 tag you can rrmove this tag

File: Kalyani/HTML/Assginment3/KalyaniA3_1.html [hunk 4]
```diff
@@ -1,57 +1,53 @@
...
-</body>
-
-</html>
\ No newline at end of file
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>
+            <h1>Assignment3_1</h1>
```
Comment (dayanandviveki) [inline Kalyani/HTML/Assginment3/KalyaniA3_1.html]: Inside title tag no need of heading tag or h1 tag

File: Kalyani/HTML/Assignment6/Mi-11-Intern.html [hunk 5]
```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (dayanandviveki) [inline Kalyani/HTML/Assignment6/Mi-11-Intern.html]: Remove Extra lines

File: Kalyani/HTML/Assignment6/Mi-11-Intern.html [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Forms</title>
+    <link rel="stylesheet" href="./Mi-11style.css" />
+</head>
+
```
Comment (dayanandviveki) [inline Kalyani/HTML/Assignment6/Mi-11-Intern.html]: chack code formmating  is in proper older

**Review by dayanandviveki — CHANGES_REQUESTED** [review]
Changes in all assignments title tag inside title tag no need of any heading tags and give proper heading as par assignment given topic and other chages also requred

**Review by dayanandviveki — APPROVED** [review]
No changes required

File: Kalyani/HTML/Assignment6/Mi-11-Intern.html [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+    <label for="Html">HTML</label>
+    <input type="checkbox" id="JavaScript" name="Skill Set" value="JavaScript" />
+    <label for="JavaScript">JavaScript</label>
+    <input type="checkbox" id="Java" name="Skill Set" value="Java" />
+    <label for="Java">Java</label>
+    <input type="checkbox" id="Oracle" name="Skill Set" value="Oracle" />
+    <label for="Oracl">Oracle</label>
+</body>
+
+</html>
```
Comment (shivnanda6272) [inline Kalyani/HTML/Assignment6/Mi-11-Intern.html]: do proper indentation, remove extra lines and add new end line.

File: Kalyani/HTML/Assignment6/Mi-11style.css [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+    text-align: right;
+    border-bottom-left-radius: 10px;
+    border-bottom-right-radius: 10px;
+    display: flex;
+    justify-content: flex-end;
+}
+
+label {
+    font-weight: bold;
+}
```
Comment (shivnanda6272) [inline Kalyani/HTML/Assignment6/Mi-11style.css]: add extra line

**Review by shivnanda6272 — CHANGES_REQUESTED** [review]
do proper indentation

File: Kalyani/HTML/Assginment1/Assin4.html [hunk 9]
```diff
@@ -1,7 +1,14 @@
 <!DOCTYPE html>
 <html>
-<head>    
-<title> <h2>Introduction</h2></title>
-<p>
-I am Kalyani Deshpande<br>
-I am a Software Enginner
+	<head>
+	    <title>
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assginment1/Assin4.html]: u dont have to add h2 tag in title write title tag in a single line and remove h2 tag

File: Kalyani/HTML/Assignment6/Mi-11-Intern.html [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+                </span>
+            <br><br>
+            <label for="Experties">Area of Experties:</label>
+            <select id="Experties" name="Experties" multiple>
+                    <option value="Software">Software</option>
+                    <option value="Networking">Networking</option>
+                    <option value="Traning">Traning</option>
+                    <option value="Markting">Markting</option>
+                    <option value="Management">Management</option>
+                </select>
```
Comment (Pawan1005) [inline Kalyani/HTML/Assignment6/Mi-11-Intern.html]: check indentation

File: Kalyani/HTML/Assignment6/Mi-11-Intern.html [hunk 11]
```diff
@@ -0,0 +1,94 @@
...
+            <label for="Resume">upload Resume:</label>
+            <input type="file" id="Resume">
+            <br><br>
+            <label for="comment">Comment</label>
+            <textarea name="Comment" id="Comment" cols="20" rows="5"></textarea>
+        </fieldset>
+        <div id="submit-b">
+         <button>Submit</button>
+        </div>
+      </body>
```
Comment (Pawan1005) [inline Kalyani/HTML/Assignment6/Mi-11-Intern.html]: check indentation

File: Kalyani/HTML/Assignment5/style.css [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+    h2 {
+        text-align: left;
+        color: black;
+    }
+    .section>div {
+        border: black solid 1px;
+    }
+    .section>div>ul>li {
+        padding: 10px;
+    }
```
Comment (Pawan1005) [inline Kalyani/HTML/Assignment5/style.css]: remove all extra spaces

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by shivnanda6272 — APPROVED** [review]

File: Kalyani/HTML/Assignment5/style.css [hunk 13]
```diff
@@ -0,0 +1,35 @@
+    body {
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assignment5/style.css]: remove extra space in file

File: Kalyani/HTML/Assignment5/KalyaniA5_a.html [hunk 14]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assignment5/KalyaniA5_a.html:2]: check the tab space it should be of four make this change in all files

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
do changes as per comments given

File: Kalyani/HTML/Assignment5/KalyaniA5_a.html [hunk 15]
```diff
@@ -0,0 +1,33 @@
...
+        <head>
+          <meta charset="UTF-8">
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assignment5/KalyaniA5_a.html]: add four tab space

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
check tab space in one file

File: Kalyani/HTML/Assignment5/KalyaniA5_a.html [hunk 16]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+		<head>
+        	<meta charset="UTF-8">
+	        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+	        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+	        <title>Assignment5</title>
+	        <link rel="stylesheet" href="./style.css">
+	    </head>
+        <body>
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assignment5/KalyaniA5_a.html]: indentation is not proper
and add four tab space only in complete file

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
check tab space in one file

File: Kalyani/HTML/Assignment5/KalyaniA5_a.html [hunk 17]
```diff
@@ -1,33 +1,33 @@
...
-            <h4><i>More About Me</i></h4>
-            <div>
-              <ul>
-                <li><a href="" target="_blank">My Resume</a></li>
-                <li><a href="" target="_blank">My Hobbies</a></li>
-                <li><a href="" target="_blank">Photo Gallery</a></li>
-              </ul>
-            </div>
-          </section>
+			<head>
```
Comment (SnehaMakade) [inline Kalyani/HTML/Assignment5/KalyaniA5_a.html]: bad indentation

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
make a proper indentation

**Review by SnehaMakade — APPROVED** [review]

File: Kalyani/HTML/Assignment6/Mi-11-Intern.html [hunk 18]
```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+        <head>
```
Comment (Pawan1005) [inline Kalyani/HTML/Assignment6/Mi-11-Intern.html]: indentation missing

File: Kalyani/HTML/Assignment6/Mi-11-Intern.html [hunk 19]
```diff
@@ -0,0 +1,94 @@
...
+                <label for="Resume">upload Resume:</label>
+                <input type="file" id="Resume">
+                <br><br>
+                <label for="comment">Comment</label>
+                <textarea name="Comment" id="Comment" cols="20" rows="5"></textarea>
+            </fieldset>
+            <div id="submit-b">
+                <button>Submit</button>
+            </div>
+        </body>
```
Comment (Pawan1005) [inline Kalyani/HTML/Assignment6/Mi-11-Intern.html]: indentation missing

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
indentation missing

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**